### PR TITLE
Bring Ubuntu current in README

### PR DESCRIPTION
**Pull request by gliptak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ steps:
 ```yaml
 jobs:
   build:
-    runs-on: ubuntu-16.04
+    runs-on: ubuntu-20.04
```
Comment (AustinShalit) [inline README.md:72]: What do you think about switching it to latest?

**Review by AustinShalit — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gliptak) [reply README.md:72]: both intents are valid. for stricter reproducible build, one might choose to set the version.

let me know if I'm to update the PR

Comment (AustinShalit) [reply README.md:72]: For a readme, I think `latest` is fine.

Comment (gliptak) [reply README.md]: updated

**Review by AustinShalit — APPROVED** [review]

**Comment by gliptak** [comment]
@AustinShalit anything else before this can be merged?

**Comment by AustinShalit** [comment]
I don't think so but I am just another person like you. I have no power here.

**Pull request closed without merging**